internal/vrf/create.go

```diff
@@ -0,0 +1,74 @@
+package vrf
+
+import (
+	"fmt"
+	"strconv"
+	"github.com/packethost/packngo"
+	"github.com/spf13/cobra"
+)
+
+
+func (c *Client) Create() *cobra.Command {
+	var metro, name, IPranges string, LocalASN int, description string
```

Comment:
In Go, the variable names should be camel-case, leading with lowercase.  The exception is when the variable is being exported (being made public).

```suggestion
	var (
	    metro string
	    name string
	    description string
	    ipRanges []string
	    localASN int
        )
```
Note: I changed ipRanges to `[]string` so it can be sent as multiple arguments to `metal-cli`: `--ip-range 1.2.3.4/24 --ip-range 2.3.4.5/24`

See https://github.com/equinix/metal-cli/blob/main/internal/devices/create.go#L34-L54 for an example.